docs/tutorial.rst

```diff
@@ -44,6 +44,9 @@ application is installed in the ``default`` namespace.
   metadata:
     name: time-logger
   spec:
+    selector:
+      matchLabels:
```

Comment:
Let's add a note  before heading `Invoking Kanister Actions`, that a better/real database example can be found by following https://github.com/kanisterio/kanister/tree/master/examples/.